src/extensions/mockFunctions.ts

```diff
@@ -0,0 +1,44 @@
+import {
+    BrsType,
+    BrsString,
+    ValueKind,
+    Callable,
+    StdlibArgument,
+    BrsInvalid,
+    RoAssociativeArray,
+} from "../brsTypes";
+import { Interpreter } from "../interpreter";
+
+export const mockFunctions = new Callable("mockFunctions", {
```

Comment:
Ok, I realize that I was the one to suggest we call this `mockFunctions` instead of `mockComponentPartial`... but given Sean's feedback and the corresponding changes, we _are_ partially mocking a component. So my vote is actually to change this back to `mockComponentPartial` (i'm the worst) 😬

Reply:
haha, now that I think about it once more it really makes sense- will revert the naming to how it was before.